testsuite/E40.py

```diff
@@ -11,10 +11,17 @@
 
 import myclass
 import foo.bar.yourclass
-#: E402
+#: Okay
 __all__ = ['abc']
 
 import foo
+__version__ = "42"
```

Comment:
@warsaw - I'm inclined to think that while __*__ could be before an import (as you opened upstream at http://bugs.python.org/issue27187), we (in PEP-8) probably don't want to mix dunder assignments between imports. What do you think?